packages/react-components/react-infobutton/src/components/InfoButton/InfoButton.tsx

```diff
@@ -7,6 +7,8 @@ import type { InfoButtonProps } from './InfoButton.types';
 
 /**
  * InfoButtons provide a way to display additional information about a form field or an area in the UI.
+ *
+ * @deprecated Use `InfoLabel` instead.
```

Comment:
can we use `@lint` jsdoc for better DX ? so when one clicks on this message they will be redirected to InfoLabel